libs/markets/src/lib/components/markets-data/markets-data.tsx

```diff
@@ -0,0 +1,21 @@
+import { useMarketNamesMap } from '../../hooks/use-market-names-map';
+import { useMarketsMap } from '../../hooks/use-markets-map';
+import { useDataProvider } from '@vegaprotocol/data-provider';
+import { marketsProvider } from '../../markets-provider';
+
+export const MarketsData = () => {
+  const updateMarketNames = useMarketNamesMap((state) => state.update);
```

Comment:
Lets just have a single store for all **static** market data. I don't see any advantage to having two stores one for names and one for everything else. It just creates a little more overhead when starting up the app

Reply:
Additionally I think we might want to wrap our app (or the page) with these start up queries and only render once they are complete. This will help simplify things by allowing us to know for certain that once we are deeper in the component tree, the market lookup _will exist_